Providers/Profile.swift

```diff
@@ -617,8 +617,10 @@ open class BrowserProfile: Profile {
     }
 
     lazy var logins: RustLogins = {
-        let sqlCipherDatabasePath = URL(fileURLWithPath: (try! files.getAndEnsureDirectory()), isDirectory: true).appendingPathComponent("logins.db").path
         let databasePath = URL(fileURLWithPath: (try! files.getAndEnsureDirectory()), isDirectory: true).appendingPathComponent("loginsPerField.db").path
+        // Though we don't migrate SQLCipher DBs anymore, we keep this call to
+        // delete any existing DBs if they still exist
+        let sqlCipherDatabasePath = URL(fileURLWithPath: (try! files.getAndEnsureDirectory()), isDirectory: true).appendingPathComponent("logins.db").path
```

Comment:
Would be good to add some error handling if possible?
I see it it's inside lazy var but want to make sure we can avoid force try!

```
//EXAMPLE
let directory: String
do {
    directory = try files.getAndEnsureDirectory()
} catch {
    // error handling
    return
}

let sqlCipherDatabasePath = URL(fileURLWithPath: directory, isDirectory: true).appendingPathComponent("logins.db").path
```

Reply:
(as discussed) unfortunately going this route ran into a bit of an issue due to the need to return `RustLogins` and given it's a `lazy var` we can't throw here easily.